### ISSUE TITLE
No skip to main content option 

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2767]
A keyboard-accessible method is provided to bypass repetitive content.

**Comment by eelzi-ONRR** [comment]
Two options in one prototype: https://5o8dme.axshare.com/?id=nouefn&p=page_1&sc=3

Click on the image/prototype and hit down key to show the expected behavior. Click on the page again and hit down key to show what happens when user tabs again.

First option is similar to https://www.irs.gov/, second option is similar to https://designsystem.digital.gov/. I'm totally fine with whichever is easier to implement - both are common and acceptable practices

This link needs to be the first thing that gets focus when the user is tabbing through the page. The link target is the first area of focus after the main navigation menu.

There's no USWDS component for this item, just some basic guidance: https://designsystem.digital.gov/components/header/

More information on this: https://webaim.org/techniques/skipnav/

It does need to have the same visible focus that other links on the page have (black outline)

**Comment by AlexandracmONRR** [comment]
USWDS has markup for this, Jeff will explore.

**Comment by greensomes** [comment]
This is not as simple as adding the markup for the USWDS feature. I believe the markup for our templates, including the main navigation menu, will need to be overhauled.

**Comment by eelzi-ONRR** [comment]
Aw bummer! I'll let y'all decide how to handle this, but since it's a global feature onrr.gov will have zero compliant pages until it's added

**Comment by greensomes** [comment]
I've added a basic skip to main content feature in [preview](https://preview-onrr-frontend.app.cloud.gov). This can be replaced with the standard USWDS markup when our template is overhauled to use USWDS.

**Comment by lpgoldstein** [comment]
@greensomes thank you! It works! @eelzi-ONRR I think it looks good.

**Comment by eelzi-ONRR** [comment]
@greensomes YESSSSSSSS! This and the focus state for everything looks perfect!!
